### DISCUSSION TITLE
Questions about library numbers.

### DISCUSSION THREAD

**Discussion by mattloose** [LooseLab/readfish#346] [Q&A]
This discussion is around the questions raised in #345 by @bioteksampath

> Thanks @alexomics

> 
> I was able to run the demo successfully. I have some additional questions regarding my own research, and I'm hoping you could provide some feedback:
> 
> 1. I want to genotype 500 structural variants (SVs) greater than 5kb across 50 lines. What would be the best approach for this?
> 1. Concerning target sizes in the target.bed file, I've heard that each target should be around 50kb to ensure good sequence coverage. Could you share your thoughts on the minimum size and maximum number of targets?
> 1. How many libraries should be maximally pooled or barcoded in a single MinION sequencing run for a genome size of 1Gb?
> 1. Could you recommend any downstream analysis tools for validation and visualization?
> 
> Apologies for the barrage of questions.

**Comment by bioteksampath** [comment]
Thanks @mattloose for creating this as a separate thread.
Yes, any thoughts on these questions would be useful for novice users.
